README.rst

```diff
@@ -133,9 +133,9 @@ Additional Links
 
 `Download Development ZIP`_  |  `Download Development TAR`_
 
-Maintained by `Adam Ginsburg`_ (`astropy.astroquery@gmail.com`_)
+Maintained by `Adam Ginsburg`_ and `Brigitta Sipocz <github.com/bsipocz>`_ (`astropy.astroquery@gmail.com`_)
 
-To cite, use our `figshare`_ DOI (http://dx.doi.org/10.6084/m9.figshare.805208) or our Zenodo DOI.
+If you use ``astroquery``, please cite our `Zenodo DOI <https://doi.org/10.5281/zenodo.1160627>`_.
```

Comment:
Keep the link to figshare here, but as a footnote or parenthetical, e.g.: (astroquery previously had a figshare DOI: URL).  We want some reference here in case a librarian comes looking to find the total historical citations of astroquery.

Reply:
Oh, OK, fair enough.